ineom/src/main/java/com/inepex/ineom/shared/validation/basicvalidators/EmailValidator.java

```diff
@@ -13,11 +13,11 @@
     private final ObjectDesc objectDesc;
 
     public final static String regExpr = "^" + "([a-zA-Z0-9]){1,40}" + // firstname
-        "(([_\\.\\-\\+])([a-zA-Z0-9]){1,40})*" + // .lastname or
+        "(([_.\\-+])([a-zA-Z0-9]){1,40})*" + // .lastname or
```

Comment:
TL;DR: Sure. The . (dot) inside a character class has no special meaning.
Quoting from https://docs.oracle.com/javase/8/docs/api/java/util/regex/Pattern.html:
"Note that a different set of metacharacters are in effect inside a character class than outside a character class. For instance, the regular expression . loses its special meaning inside a character class, while the expression - becomes a range forming metacharacter."